Objective-C/CBLDatabase.h

```diff
@@ -364,7 +364,7 @@ __deprecated_msg("Use [[database defaultCollection] purgeDocumentWithID:error:]
 
 
 /** 
- Adds a database change listener. Changes will be posted on the main queue.
+ Adds a database change listener to listen to changes in the default collection. Changes will be posted on the main queue.
```

Comment:
Is this still called a "database change listener"?  In Java-land, they are now called CollectionChangeListeners

Reply:
I will remove the word "database" to avoid confusion. I think Java does that here :

https://github.com/couchbase/couchbase-lite-java-common/blob/master/common/main/java/com/couchbase/lite/AbstractDatabase.java#L822-L835